cfme/tests/infrastructure/test_publish_vm_to_template.py

```diff
@@ -19,8 +20,15 @@ def vm_crud(provider):
     collection = provider.appliance.provider_based_collection(provider)
     vm_name = random_vm_name(context='pblsh')
     vm = collection.instantiate(vm_name, provider)
-    deploy_template(vm.provider.key, vm_name, provider.data['small_template'],
-                    timeout=2500)
+    try:
+        if provider.one_of(VMwareProvider):
+            deploy_template(vm.provider.key, vm_name,
+                provider.data['templates']['small_template']['name'], timeout=2500)
+        else:
```

Comment:
This looks like yaml inconsistency that should be fixed in the yaml.

Reply:
Will open up a yamls PR.